client/sections.js

```diff
@@ -42,4 +42,16 @@ sections.push( {
 	enableLoggedOut: true,
 } );
 
+if ( config.isEnabled( 'hello-world' ) ) {
+	sections.push( {
+		name: 'hello-world',
+		paths: [ '/hello-world' ],
+		module: 'hello-world',
+		enableLoggedOut: true,
+		secondary: false,
```

Comment:
Would  someone know to point out what this changes and where it is used?

Searching the codebase just for a term `secondary` proved tricky and I couldn't find it by following the code logic. :-)

Reply:
@tyxla explained this well at Slack:

> it’s supposed to hold the sidebar, so in the section config it basically instructs whether you’re expecting to have a sidebar or not
> anyway, we’re setting up the sidebars later in the controllers where necessary, so currently this will only affect > the loading page in most cases (whether the loading page will have a sidebar or not)
> the loading page - this is what you see briefly when you reload Calypso on a certain page
> if you want to give it a try:
> - enable `secondary` for the `jetpack-connect` section
> - rebuild
> - load http://calypso.localhost:3000/jetpack/connect and you’ll see a sidebar in the loading page
> - now load https://wordpress.com/jetpack/connect and you’ll see no sidebar in the loading page

Thanks!